releasenotes-builder/pom.xml

```diff
@@ -81,6 +81,12 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
```

Comment:
Out of curiosity. What is your stance on adding dependencies to checkstyle projects?

Reply:
We don't want to be crazy with dependencies, but if there is an actual need and no other way then we can discuss and include new ones. This dependency is not in main repo as it moved to junit 5 but we are using junit 4 here. Before main repo's upgrade, we use to have this dependency and similar code. I just copied things from the old version of the main repo.